src/core/scene/a-scene.js

```diff
@@ -270,13 +270,14 @@ class AScene extends AEntity {
    * @param {bool?} useAR - if true, try immersive-ar mode
    * @returns {Promise}
    */
-  enterVR (useAR) {
+  enterVR (useAR, useOfferSession = false) {
     var self = this;
     var vrDisplay;
     var vrManager = self.renderer.xr;
     var xrInit;
 
     // Don't enter VR if already in VR.
+    if (navigator.xr.offerSession === undefined) { return Promise.resolve('OfferSession is not supported.'); }
```

Comment:
Should we check the useOfferSession parameter here as well?

Reply:
fixed